components/test_raftstore/src/cluster.rs

```diff
@@ -1411,6 +1411,28 @@ impl<T: Simulator> Cluster<T> {
             .unwrap();
     }
 
+    pub async fn call_prepare_flashback(&mut self, region_id: u64, store_id: u64) {
```

Comment:
Maybe should rename it to `call_and_wait_prepare_flashback` to indicate the potential blocking.

Reply:
get it.